cli/tests/environment-install.bats

```diff
@@ -167,3 +167,52 @@ teardown() {
 @test "i6: install on a pushed environment stages locally" {
   skip "remote environments handled in another phase"
 }
+
+@test "'flox install' installs by path" {
+  run "$FLOX_BIN" init;
+  assert_success;
+  run "$FLOX_BIN" install hello;
+  assert_success;
+  manifest=$(cat "$PROJECT_DIR/.flox/env/manifest.toml");
+  # This also checks that it correctly infers the install ID
+  assert_regex "$manifest" 'hello.path = "hello"';
+}
+
+@test "'flox install' infers install ID" {
+  run "$FLOX_BIN" init;
+  assert_success;
```

Comment:
Nit: we don't actually need `run` or to `assert` here. In the other tests went with only asserting the command under test.
I don't think there's anything _wrong_ with it.

```suggestion
  "$FLOX_BIN" init;
```

Reply:
I don't want to proceed with the test if something else has gone wrong.